test/pdftotext.test.js

```diff
@@ -24,7 +24,7 @@ describe('pdftotext', function(){
     it('should throw exception', function(){
       var ptext = new pdftotext(__dirname + '/pdfs/invalidfile.pdf');
       function fn() {
-        var stdout = ptext.getTextSync();
+        return ptext.getTextSync().toString('utf8');
       }
       assert.throws(fn, /I\/O Error/);
```

Comment:
@lirantal Still failing looks like we need to make it more lenient. Lets try assert.throws(fn, /error/gi)